docs/Custom-brimcap-load-Config.md

```diff
@@ -0,0 +1,434 @@
+
+1. The Brimcap-bundled Zeek includes the additional packages
+[geoip-conn](https://github.com/brimdata/geoip-conn),
+[zeek-community-id](https://github.com/corelight/zeek-community-id),
+[HASSH](https://github.com/salesforce/hassh),
+and [JA3](https://github.com/salesforce/ja3). These would typically be
+installed via [Zeek Package Manager](https://docs.zeek.org/projects/package-manager/en/stable/quickstart.html).
+
+2. The Brimcap-bundled Suricata includes a
+[YAML configuration](https://github.com/brimdata/build-suricata/blob/master/brim-conf.yaml)
+that (among other things) enables the `community_id` field, which is essential
+for joining to the `community_id` field in Zeek events that may give context
+about why the alert fired. This is achieved in the bundled Suricata YAML
+configuration by setting
+[`community-id: yes`](https://github.com/brimdata/build-suricata/blob/853fab6d7c21325f57e113645004b1107b78d840/brim-conf.yaml#L51-L52) and JSON
+[`escape-slash: no`](https://github.com/brimdata/build-suricata/blob/853fab6d7c21325f57e113645004b1107b78d840/brim-conf.yaml#L80-L81)
+for the `eve-log` output.
+
+3. To ensure [rules](https://suricata.readthedocs.io/en/suricata-6.0.0/rules/)
+are kept current, the Brim app invokes the bundled "Suricata Updater" once
+each time it is opened. However, in a custom configuration, no attempt is made
+to trigger updates on your behalf. You may choose to to periodically run your
+`suricata-update` manually or consider a scheduled mechanism such as `cron`.
+
+## Example Configuration
+
+For use with the Zeek/Suricata we just installed, a sample
+[Brimcap YAML configuration](https://github.com/brimdata/brimcap/blob/main/examples/zeek-suricata.yml)
+and accompanying wrapper scripts for [Zeek](https://github.com/brimdata/brimcap/blob/main/examples/zeek-wrapper.sh)
+and [Suricata](https://github.com/brimdata/brimcap/blob/main/examples/suricata-wrapper.sh)
+are available in the Brimcap repo. If Brim is running and all scripts/binaries
+invoked are in the `$PATH`, the configuration can be tested outside the app to
+import a `sample.pcap` like so:
+
+```
+$ /opt/Brim/resources/app.asar.unpacked/zdeps/zed api new testpool
+$ /opt/Brim/resources/app.asar.unpacked/zdeps/brimcap load -root "$HOME/.config/Brim/data/brimcap-root" -config zeek-suricata.yml -p testpool sample.pcap
+```
+
+* **Note**: The `zdeps` directory that contains the `zed` and `brimcap`
+binaries varies per platform.
+
+|**OS Platform**|**Location**|
+|---------------|------------|
+| **Windows**   | `%USERPROFILE%\AppData\Local\Programs\Brim\resources\app.asar.unpacked\zdeps` |
+| **macOS**     | `/Applications/Brim.app/Contents/Resources/app.asar.unpacked/zdeps` |
+| **Linux**     | `/opt/Brim/resources/app.asar.unpacked/zdeps` |
+
+If successful, the new pool will appear in Brim, allowing you to browse the
+logs and open flows from the pcap via the **Packets** button.
+
+![NetFlow Pool](media/Custom-Zeek-Suricata-Pool.png)
```

Comment:
Note that for now we're still doing **View > Reload** manually in the app to see new pool data that has been posted from outside the app. brimdata/brim#950 is tracking the enhancement here.

Reply:
So that I can release this article to beta testers, for now I've put a "Note:" block in the article linking to brimdata/brim#950 so users know to click **View > Reload** manually for now.